compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -110,6 +115,10 @@ interface PlatformContext {
     val parentFocusManager: FocusManager get() = EmptyFocusManager
     fun requestFocus(): Boolean = true
 
+    fun createPlatformDragAndDropManager(): PlatformDragAndDropManager {
```

Comment:
```suggestion
    val dragAndDropManager: PlatformDragAndDropManager
```
- to be consistent it should be a property, and creation/returning existing are implementation details. Unless it is stateful, and we want a new state every time.
- the name shouldn't contain `platform`, as the owner class already contains it.

Reply:
> The idea behind "Empty" is to provide a maximum basic/"empty" platform that doesn't crash, even if user call this API directly.
> Otherwise, users can encounter crashes if they test with the default "ComposeScene" a widget/screen that contains dragAndDropSource.

As I am implementing drag-and-drop for desktop only, and the code beforehand was throwing an "unimplemented" exception, I wanted to preserve that on platforms other than desktop.

I now return an "Empty" implementation in `PlatformContext`, and an "Unimplemented" implementation in the iOS and web `PlatformContext`s.

Reply:
> to be consistent it should be a property, and creation/returning existing are implementation details. Unless it is stateful, and we want a new state every time.

It is stateful and a new instance is needed every time. Several `RootNodeOwner`s can share the same `PlatformContext`, but each `RootNodeOwner` needs its own `DragAndDropManager`.

Reply:
> the name shouldn't contain platform, as the owner class already contains it.

I think that's confusing even more, but ok.